domain-server/resources/web/js/shared.js

```diff
@@ -51,17 +51,17 @@ $.extend(Settings, {
 var URLs = {
   // STABLE METAVERSE_URL: https://metaverse.highfidelity.com
   // STAGING METAVERSE_URL: https://staging.highfidelity.com
-  METAVERSE_URL: 'https://metaverse.highfidelity.com',
-  CDN_URL: 'https://cdn.highfidelity.com',
-  PLACE_URL: 'https://hifi.place',
+  DEFAULT_METAVERSE_URL: "https://metaverse.vircadia.com/live",
+  CDN_URL: 'https://content.vircadia.com/eu-c-1',
+  PLACE_URL: 'https://hifi.place'
```

Comment:
True, I don't know if we have a proper replacement for that yet since I'm not entirely sure what the previous... one did. I guess it was a webpage that redirected you to the actual metaverse link? I have a domain-name ready to go to replace this one, I just am unsure of the functionality.

This was used in the Wizard only.

Reply:
Searching for the URL finds it in:
```
domain-server\resources\web\js\shared.js
domain-server\resources\web\wizard\js\wizard.js
```
It's to do with place names. Could perhaps try the wizard out while HiFi servers are still up and running, to see.

Reply:
hifi.place/fdafiojfoi shows the example functionality. For now I can fill in a placeholder URL but I'll setup a page for that later.